inspirehep/dojson/hepnames/fields/bd1xx.py

```diff
@@ -464,7 +469,9 @@ def advisors2marc(self, key, value):
     """The advisors for all types of degrees."""
     return {
         'a': value.get('name'),
-        'g': value.get('_degree_type')
+        'g': value.get('_degree_type'),
+        'x': get_recid_from_ref(value.get('record')),
+        'y': '1' if value.get('curated_relation') else None
```

Comment:
Just found http://inspirehep.net/record/1019112/export/xme and it seems that `y` disappears if there's no link via recid or INSPIRE-ID.